ci-operator/config/openshift/appliance/openshift-appliance-master.yaml

```diff
@@ -44,6 +44,13 @@ tests:
     make lint
   container:
     from: test-bin
+- as: publish
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: agent-preinstall-image-builder
```

Comment:
can we call it 'openshift-appliance' on quay?

Reply:
Added `IMAGE_REPO` to try to change that. Also changed temporarily the job to pre-submit to be able to rehearse it (thanks for that tip @osherdp).

Reply:
The temporary pre-submit job rehearsal worked. Changed back to post-submit. Please check agin @danielerez.